dask_expr/tests/test_distributed.py

```diff
@@ -86,3 +87,16 @@ async def test_merge_p2p_shuffle(c, s, a, b):
     lib.testing.assert_frame_equal(
         x.reset_index(drop=True), df_left.merge(df_right)[["b", "c"]]
     )
+
+
+def test_sort_values():
+    with LocalCluster() as cluster:
```

Comment:
Any way to leverage `gen_cluster` here?

Reply:
No, it's async

Reply:
Can you explain more? Also, do we really want to allow a test to use all the available cores on our machine like this?

Reply:
You maybe want these fixtures from distributed.utils_test

```python
@pytest.fixture
def cluster_fixture(loop):
    with cluster() as (scheduler, workers):
        yield (scheduler, workers)


@pytest.fixture
def s(cluster_fixture):
    scheduler, workers = cluster_fixture
    return scheduler


@pytest.fixture
def a(cluster_fixture):
    scheduler, workers = cluster_fixture
    return workers[0]


@pytest.fixture
def b(cluster_fixture):
    scheduler, workers = cluster_fixture
    return workers[1]


@pytest.fixture
def client(loop, cluster_fixture):
    scheduler, workers = cluster_fixture
    with Client(scheduler["address"], loop=loop) as client:
        yield client
```

They handle cleanup and various sanity checks. 

Alternatively, if you want things to be very fast, you might also want `with LocalCluster(processes=False)`

Reply:
the gen_cluster decorator creates a async cluster and async tests will fail for sort_values/set_index because of intermediate computes. That's not a dask-expr problem, dask/dask has the same issue. 

Sure, we can restrict the number of workers

Reply:
I think I prefer the fast version, this is only supposed to test that the function serialisation works as expected

Reply:
>Sure, we can restrict the number of workers

Sorry for mentioning that - I care less about that detail (for now).

Reply:
See https://github.com/dask/distributed/pull/8167 This one has the same issue in dask/dask